source/_integrations/weather.markdown

```diff
@@ -112,42 +103,60 @@ The response data field `forecast` is a list of forecasted conditions at a given
 | `precipitation_probability` | The probability of precipitation in %. | 0
 | `precipitation` | The precipitation amount in the unit indicated by the `precipitation_unit` state attribute. | 0
 | `pressure` | The air pressure in the unit indicated by the `pressure_unit` state attribute. | 1019
-| `temperature` | The temperature in the unit indicated by the `temperature_unit` state attribute. If templow is also provided, this is the higher temperature. | 14.2
+| `temperature` | The temperature in the unit indicated by the `temperature_unit` state attribute. If `templow` is also provided, this is the higher temperature. | 14.2
 | `templow` | The lower temperature in the unit indicated by the `temperature_unit` state attribute. | 5.0
 | `uv_index` | The UV index. | 3
 | `wind_bearing` | The wind bearing in azimuth angle (degrees) or 1-3 letter cardinal direction. | 268
 | `wind_gust_speed` | The wind gust speed in the unit indicated by the `wind_speed_unit` state attribute. | 34.41
 | `wind_speed` | The wind speed in the unit indicated by the `wind_speed_unit` state attribute. | 24.41
 
 
-Example forecast:
+{% details "Example service response" %}
```

Comment:
I'd also add a simple example script or template sensor where the service is called and the response variable is accessed in a template. Not required for this PR though.

Reply:
Takes just a minute and is a good addition.